psi4/driver/p4util/util.py

```diff
@@ -138,13 +138,12 @@ def set_memory(inputval):
         units = ''
     # Handle memory given as a string
     if isinstance(inputval, basestring):
```

Comment:
Changing this line to a simple `else:` would fix that, and should still work for unicode or raw strings as well